client/blocks/reader-author-link/index.jsx

```diff
@@ -29,7 +29,7 @@ const ReaderAuthorLink = ( { author, post, siteUrl, children } ) => {
 	}
 
 	return (
-		<ExternalLink className="reader-author-link" href={ linkUrl } target="_blank" onClick={ recordAuthorClick }>
+		<ExternalLink className="reader-author-link" href={ linkUrl } target="_blank" rel="noopener noreferrer" onClick={ recordAuthorClick }>
```

Comment:
The implementation of `<ExternalLink />` is such that `rel` will be overridden and remove the `noopener noreferrer`:

https://github.com/Automattic/wp-calypso/blob/dcf8306/client/components/external-link/index.jsx#L41

To that end, if the purpose of external link is for linking externally, should we just add these values to the component itself?